composer.json

```diff
@@ -22,17 +22,24 @@
     },
     "require-dev": {
         "composer/installers": "^1.9",
-        "johnpbloch/wordpress": "^5.4",
+        "johnpbloch/wordpress": "^6.1",
```

Comment:
Do we need to support a lower version? If so, we may need to downgrade `phpunit`.

When using any version lower than `6.1` here, an error is encountered in the test run because of the renaming of `wp-db.php` --> `class-wpdb.php` here:

<img width="1072" alt="image" src="https://github.com/missionmike/wp-graphql-testcase/assets/4269287/dc84d510-f0a6-4d91-b227-39669406d0b3">

See: https://wordpress.org/support/topic/file-wp-db-php-is-deprecated-since-version-6-1-0/